packages/babel-plugin-transform-wpcalypso-async/jest.config.js

```diff
@@ -1 +1,5 @@
-module.exports = { preset: '@automattic/calypso-build', rootDir: __dirname };
+module.exports = {
```

Comment:
This can't be changed in `test/packages/jest.config.js` :(

When using `projects` (as `test/packages/jest.config.js` does) to specify Jest subprojects, it will use the config form the file `jest.config.js` in those subprojects (i.e. this file). If it doesn't define a `cacheDirectory`, it won't pick the value from the "parent" project. Instead, it will use a temporary dir.

Therefore every subproject that is referenced via `projects` has to define its own `cacheDirectory`.